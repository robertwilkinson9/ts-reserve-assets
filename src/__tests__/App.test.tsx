import { describe, it } from 'vitest';
//import { renderToString } from 'react-dom/server';
//import { App } from '../App';

describe('Renders main page correctly', async () => {
    it('Should render the page correctly', async () => {
//      const app_string = await renderToString(<App />);
//      expect(app_string).toMatchSnapshot();
    });
});
