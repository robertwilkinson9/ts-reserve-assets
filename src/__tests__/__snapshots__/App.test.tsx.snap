// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Renders main page correctly > Should render the page correctly 1`] = `"<hr data-testid=\\"header_ruler\\"/><div role=\\"group\\" class=\\"chakra-form-control css-1kxonj9\\"><label data-testid=\\"formlabel_3\\" id=\\"field-:Rbq:-label\\" for=\\"field-:Rbq:\\" class=\\"chakra-form__label css-g6pte\\">interesting Form <!-- -->Three</label><div id=\\"field-:Rbq:-helptext\\" class=\\"chakra-form__helper-text css-186pyma\\">We&#x27;ll never share your Form <!-- -->Three<!-- --> data.</div><div class=\\"chakra-stack css-1v8my8o\\"><label data-testid=\\"calendar_label\\">Start DateTime</label><div data-testid=\\"calendar_datepicker\\"><div class=\\"react-datepicker-wrapper\\"><div class=\\"react-datepicker__input-container\\"><input type=\\"text\\" class=\\"\\" value=\\"8/03/2024 09:11 AM\\"/></div></div></div><label data-testid=\\"calendar_label\\">End DateTime</label><div data-testid=\\"calendar_datepicker\\"><div class=\\"react-datepicker-wrapper\\"><div class=\\"react-datepicker__input-container\\"><input type=\\"text\\" class=\\"\\" value=\\"\\"/></div></div></div><div data-testid=\\"bucket_label_div\\" class=\\"row\\" id=\\"pull-left\\"><div class=\\"col-sm-12 align-items-left\\"><label data-testid=\\"bucket_label_label\\" class=\\"mb-0 font-weight-bold flabel\\">Author</label></div></div><div data-testid=\\"bucket_div\\" class=\\"container\\"><div data-testid=\\"bucket_button_checked\\" class=\\"row\\" id=\\"bucket_radios\\"><div class=\\"col\\"><input data-testid=\\"bucket_button_checked_input\\" type=\\"radio\\" id=\\"Jane Austen\\" name=\\"bucket\\" checked=\\"\\" value=\\"0\\"/><label data-testid=\\"bucket_button_checked_label\\" for=\\"Jane Austen\\">Jane Austen</label></div></div><div data-testid=\\"bucket_button_unchecked\\" class=\\"row\\" id=\\"bucket_radios\\"><div class=\\"col\\"><input data-testid=\\"bucket_button_unchecked_input\\" type=\\"radio\\" id=\\"George Orwell\\" name=\\"bucket\\" value=\\"1\\"/><label data-testid=\\"bucket_button_unchecked_label\\" for=\\"George Orwell\\">George Orwell</label></div></div></div><div data-testid=\\"items_div\\" id=\\"itemPulldown\\"><label data-testid=\\"items_label\\" class=\\"mb-0 font-weight-bold\\">Book</label><div class=\\"chakra-select__wrapper css-42b2qy\\"><select id=\\"field-:Rbq:\\" class=\\"chakra-select css-161pkch\\"><option value=\\"\\">Select option</option><option value=\\"Emma\\">Emma</option><option value=\\"Mansfield Park\\">Mansfield Park</option><option value=\\"Northanger Abbey\\">Northanger Abbey</option><option value=\\"Persuasion\\">Persuasion</option><option value=\\"Pride and Prejudice\\">Pride and Prejudice</option><option value=\\"Sense and Sensibility\\">Sense and Sensibility</option></select><div class=\\"chakra-select__icon-wrapper css-iohxn1\\"><svg viewBox=\\"0 0 24 24\\" role=\\"presentation\\" class=\\"chakra-select__icon\\" focusable=\\"false\\" aria-hidden=\\"true\\" style=\\"width:1em;height:1em;color:currentColor\\"><path fill=\\"currentColor\\" d=\\"M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z\\"></path></svg></div></div></div><div class=\\"css-0\\"><div data-testid=\\"emailaddress\\" id=\\"emailaddress\\"><div role=\\"group\\" class=\\"chakra-form-control css-1kxonj9\\"><label data-testid=\\"emailaddress_label\\" for=\\"email\\" id=\\"field-:R2mtbq:-label\\" class=\\"chakra-form__label css-g6pte\\">Email address</label><input type=\\"email\\" placeholder=\\"enter Email address\\" id=\\"email\\" class=\\"chakra-input css-10m69nq\\" value=\\"\\"/><div id=\\"field-:R2mtbq:-helptext\\" class=\\"chakra-form__helper-text css-186pyma\\">We&#x27;ll never share your email.</div></div></div></div><button type=\\"button\\" class=\\"chakra-button css-ez23ye\\">Reserve book<!-- --> </button></div></div><span></span><span id=\\"__chakra_env\\" hidden=\\"\\"></span>"`;
